src/tests/minicore.md

```diff
@@ -2,7 +2,7 @@
 
 <!-- date-check Oct 2024 -->
 
-[`tests/auxiliary/minicore.rs`][`minicore`] is a test auxiliary for
+[`minicore`] is a test auxiliary for
```

Comment:
I would like keeping the more full path

Reply:
Yeah it would help to spell out the path in the first sentence to make it clear what this is referring to, but later ones can just use the shorthand link :+1:

Reply:
Done.